Rubberduck.Parsing/VBA/DeclarationCaching/DeclarationFinder.cs

```diff
@@ -1177,6 +1177,16 @@ public IEnumerable<IdentifierReference> IdentifierReferences(QualifiedModuleName
                 : Enumerable.Empty<IdentifierReference>();
         }
 
+        /// <summary>
+        /// Gets all identifier references in the specified module.
```

Comment:
nit: isn't it the `selection` that's specified? xml-doc seems to contradict the code here.

Reply:
Classic copy-paste error; will fix.